src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -2,36 +2,42 @@
   "iris.sodium.failure.title": "Iris konnte nicht geladen werden!",
   "iris.sodium.failure.download": "Sodium herunterladen",
   "iris.sodium.failure.reason.notFound": "Iris erfordert Sodium für optimale Leistung, aber Sodium ist nicht installiert. Bitte lade Sodium herunter, füge es deinem Mods-Ordner hinzu und starte das Spiel neu.",
   "iris.sodium.failure.reason.incompatible": "Iris erfordert Sodium für optimale Leistung, aber es wurde eine inkompatible Version gefunden. Bitte lade die passende Version von Sodium herunter, füge sie deinem Mods-Ordner hinzu, entferne die falsche Version und starte das Spiel neu.",
 
   "options.iris.apply": "Anwenden",
   "options.iris.refresh": "Aktualisieren",
-  "options.iris.openShaderPackFolder": "Shaderpaket-Ordner öffnen ...",
-  "options.iris.shaderPackSettings": "Shaderpaket-Einstellungen ...",
-  "options.iris.shaderPackList": "Shaderpaket-Liste ...",
-  "options.iris.refreshShaderPacks": "Shaderpakete aktualisieren",
+  "options.iris.openShaderPackFolder": "Shader-Ordner öffnen ...",
+  "options.iris.shaderPackSettings": "Shader-Einstellungen ...",
+  "options.iris.shaderPackList": "Shader-Liste ...",
+  "options.iris.refreshShaderPacks": "Shader/Liste aktualisieren",
   "options.iris.shaderPackSelection": "Shaderpakete ...",
   "options.iris.shaderPackSelection.title": "Shaderpakete",
-  "options.iris.shaderPackSelection.addedPack": "Shaderpaket hinzugefügt \"%s\"",
+  "options.iris.shaderPackSelection.addedPack": "Shaderpaket \"%s\" hinzugefügt",
   "options.iris.shaderPackSelection.addedPacks": "%s Shaderpakete hinzugefügt",
-  "options.iris.shaderPackSelection.failedAdd": "Die Dateien waren keine gültigen Shaderpakete.",
-  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" ist kein gültiges Shaderpaket",
-  "options.iris.shaderPackSelection.copyError": "Konnte Shaderpaket nicht hinzufügen \"%s\"",
-  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" befindet sich bereits im Shaderpaket-Ordner!",
+  "options.iris.shaderPackSelection.failedAdd": "Die Dateien waren keine zulässigen Shaderpakete",
+  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" ist kein zulässiges Shaderpaket",
+  "options.iris.shaderPackSelection.copyError": "Konnte Shaderpaket \"%s\" nicht hinzufügen",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" befindet sich bereits im Shader-Ordner!",
+  "options.iris.shaderPackOptions.tooManyFiles": "Es können nicht mehrere Shader-Einstellungsdateien importiert werden!",
+  "options.iris.shaderPackOptions.failedImport": "Shader-Einstellungsdatei \"%s\" konnte nicht importiert werden",
+  "options.iris.shaderPackOptions.importedSettings": "Shader-Einstellungen aus \"%s\" importiert",
   "options.iris.shaders.disabled": "Shader: Deaktiviert",
   "options.iris.shaders.enabled": "Shader: Aktiviert",
+  "options.iris.shaders.nonePresent": "Shader: Keine Shader vorhanden",
```

Comment:
I find this a little confusing

Reply:
Yeah, this one is a little weird but I don’t like the idea of a “Paket” in general so I will remove the second Shader here

Reply:
Sounds good